root/io/filemerger/execTestMultiMerge.C

```diff
@@ -70,19 +70,22 @@ int testSimpleFile(const char *filename, Long64_t entries, Int_t compSetting, Lo
 int execTestMultiMerge()
 {
    Int_t result = 0;
-   int hsimpleFTolerance = 6; // 5 for non fst builds
+   int hsimpleFTolerance = 10; // 5 for non fst builds
```

Comment:
Do we have a way to detect if were are in a 'fst' build?

Reply:
@pcanal it could defined R__FAST_MATH, I will push new version of fix

Reply:
@pcanal sorry, I mistaken hsimpleFTolerance is for hsimpleF.test case (it is failing only on not-fast aarch64 native compiler build (gcc4.8.5))